azure/resource-containers/base-az-resource-container/base-az-resource-container.psm1

```diff
@@ -1,5 +1,5 @@
-$Public  = @( Get-ChildItem -Path $PSScriptRoot\functions\public\*.ps1 -ErrorAction SilentlyContinue )
-$Private = @( Get-ChildItem -Path $PSScriptRoot\functions\private\*.ps1 -ErrorAction SilentlyContinue )
```

Comment:
By removing these lines, I no longer need a try catch to catch alot of errors now.